turbinia/workers/analysis/jenkins.py

```diff
@@ -37,7 +37,7 @@ class JenkinsAnalysisTask(TurbiniaTask):
   TASK_CONFIG = {
       # This is the length of time in seconds that the collected passwords will
       # be bruteforced.
-      'bruteforce_timeout': 300
+      'bruteforce_timeout': 600
```

Comment:
Just to confirm, with the existing config and password list this will be long enough to pick up `jenkins` then?

Reply:
Yep